Mathlib/Algebra/Order/Sum.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2024 Martin Dvorak. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Martin Dvorak
+-/
+import Mathlib.Order.Basic
+import Mathlib.Algebra.Group.Pi.Basic
+
+/-!
+# Interaction between `Sum.elim` and `≤`
+
+This file provides basic API for part-wise comparison of `Sum.elim` vectors.
+-/
+
+namespace Sum
+
+variable {α₁ α₂ β : Type*} [LE β]
+
+@[simp]
+lemma elim_le_elim_iff (u₁ v₁ : α₁ → β) (u₂ v₂ : α₂ → β) :
+    Sum.elim u₁ u₂ ≤ Sum.elim v₁ v₂ ↔ u₁ ≤ v₁ ∧ u₂ ≤ v₂ :=
+  Sum.forall
```

Comment:
This has nothing to do with algebra, hence shouldn't be under` Algebra`. Please move it to `Order.Basic`

Reply:
Did you see my comment above? The file with current imports does not know ordering on vectors.

Reply:
`Pi.le_def` is in `Order.Basic` so I don't believe you.

Reply:
Oh, my bad! I was putting it too high in the file. Sorry for the misinformation.